source/renderer/app/api/utils/patchAdaApi.js

```diff
@@ -28,8 +28,13 @@ export default (api: AdaApi) => {
       const networkInfo: NetworkInfoResponse = await getNetworkInfo(api.config);
       Logger.debug('AdaApi::getNetworkInfo (PATCHED) success', { networkInfo });
 
-      /* eslint-disable-next-line camelcase */
-      const { sync_progress, node_tip, network_tip, next_epoch } = networkInfo;
+      const {
+        sync_progress, // eslint-disable-line camelcase
+        node_tip, // eslint-disable-line camelcase
+        network_tip, // eslint-disable-line camelcase
+        next_epoch, // eslint-disable-line camelcase
+        future_epoch, // eslint-disable-line camelcase
```

Comment:
@daniloprates please revert this part as we don't get `future_epoch` from the API.

Reply:
✅